valkey.conf

```diff
@@ -1915,29 +1915,57 @@ aof-timestamp-enabled no
 # cluster-announce-port 0
 # cluster-announce-bus-port 6380
 
-################################## SLOW LOG ###################################
-
-# The server Slow Log is a system to log queries that exceeded a specified
-# execution time. The execution time does not include the I/O operations
-# like talking with the client, sending the reply and so forth,
-# but just the time needed to actually execute the command (this is the only
-# stage of command execution where the thread is blocked and can not serve
-# other requests in the meantime).
-#
-# You can configure the slow log with two parameters: one tells the server
+################################## COMMAND LOG ###################################
+
+# The Command Log system is used to record commands that consume significant resources
+# during server operation, including CPU, memory, and network bandwidth.
+# These commands and the data they access may lead to abnormal instance operations,
+# the commandlog can help users quickly and intuitively locate issues.
+# Currently, there are three types of commandlogs supported: those used to record
+# commands with long execution times, those with large requests, and those with
+# large query replies.
+
+# The first type is SLOW, log queries that exceeded a specified execution time.
+# The execution time does not include the I/O operations like talking with the client,
+# sending the reply and so forth, but just the time needed to actually execute the
+# command (this is the only stage of command execution where the thread is blocked
+# and can not serve other requests in the meantime).
+#
+# You can configure the slow threshold with two parameters: one tells the server
 # what is the execution time, in microseconds, to exceed in order for the
 # command to get logged, and the other parameter is the length of the
 # slow log. When a new command is logged the oldest one is removed from the
 # queue of logged commands.
-
+#
 # The following time is expressed in microseconds, so 1000000 is equivalent
 # to one second. Note that a negative number disables the slow log, while
 # a value of zero forces the logging of every command.
-slowlog-log-slower-than 10000
-
+commandlog-execution-slower-than 10000
```

Comment:
@soloestoy Let's mention the configs in the top comment.

I see here not only slowlog command is now an alias but also the slowlog configs. Shall we mention the slowlog configs in valkey.conf and describe they are now aliases for the new configs?

When I see also configs are changed into aliases, it makes me think deprecated is maybe the right term. But we can mention that the slowlog configs still work and we can list them here too. We can also say that there is no plan to remove to them?